cpp/ps0001/problem 2.cpp

```diff
@@ -1,11 +1,17 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
-int number1,number2,number3;
-cout << "Please Enter 3 numbers \n" << endl;
-cin >> number1>>number2>>number3;
-int sum=(number1+number2+number3);
- cout <<" sum equal\n "<<sum<<" \n";
-return 0;
-}
\ No newline at end of file
+  // prompting user for 3 numbers to add 1/2
+  unsigned short number1, number2, number3;
+  cout << "Please Enter 3 numbers \n" << endl;
+  cin >> number1 >> number2 >> number3;
+
+  // outputting sum of the 3 numbers 3/3
+  unsigned short = (number1 + number2 + number3);
+  cout << " sum equal\n " << sum << " \n";
+  
+  return 0;
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code